### PR TITLE
Skip testWindowMaximizeBeforePageVisit when using xvfb

### PR DESCRIPTION
**Pull request by peterrehm**
Needed when minkphp/driver-testsuite#10 gets merged.

### CODE REVIEW

File: tests/Selenium2Config.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ public function skipMessage($testCase, $test)
 
         if (
             'Behat\Mink\Tests\Driver\Js\WindowTest' === $testCase
-            && 'testWindowMaximize' === $test
+            && ('testWindowMaximize' === $test || 'testWindowMaximizeBeforePageVisit' === $test)
```
Comment (aik099) [inline tests/Selenium2Config.php]: Or maybe you can check, that test name starts with `testWindowMaximize`.

Comment (peterrehm) [inline tests/Selenium2Config.php]: Shall I change as follows?

````php
0 === strpos($test, 'testWindowMaximize')
````

**Comment by peterrehm** [comment]
I updated it as per your remark

**Comment by aik099** [comment]
Merging, thanks @peterrehm .

**Comment by peterrehm** [comment]
Thanks. 👍

**Pull request merged**